src/verifier/tdx/mod.rs

```diff
@@ -138,22 +134,20 @@ fn generate_parsed_claim(
     match cc_eventlog {
         Some(ccel) => {
             match (
-                ccel.query_digest(MeasuredEntity::TdPayload),
-                ccel.query_event_data(MeasuredEntity::TdPayload),
+                ccel.query_digest(MeasuredEntity::TdShimKernal),
+                ccel.query_event_data(MeasuredEntity::TdShimKernal),
             ) {
-                (Some(td_paylod_digest), Some(event_data)) => {
+                (Some(kernal_digest), Some(event_data)) => {
                     let uefi_platform_firmware_blob2 =
                         ParsedUefiPlatformFirmwareBlob2::try_from(event_data)
                             .map_err(|e| anyhow!("Parse td_payload event data failed: {:?}", e))?;
                     let digest_name = format!(
                         "tdx-kernel-size{:?}",
                         uefi_platform_firmware_blob2.blob_length
                     );
-                    claim_map.insert(digest_name, serde_json::Value::String(td_paylod_digest));
-                }
-                _ => {
-                    warn!("parse CC EventLog: There is no tdx kernel measurement entry")
+                    claim_map.insert(digest_name, serde_json::Value::String(kernal_digest));
                 }
+                _ => todo!(),
```

Comment:
there is a `panic!` behind `todo!`, maybe we should use a `warn!` or something else here?